src/server/game/Miscellaneous/SharedDefines.h

```diff
@@ -1061,6 +1061,12 @@ enum Language
 
 #define LANGUAGES_COUNT   31
 
+enum FactionSelection
```

Comment:
Feels weird sticking it in SharedDefines

Reply:
Any preferable location?

Reply:
I'd have suggested one if I knew, but since this is straight out of an opcode, I'm really not sure

Reply:
Actually you don't need it anywhere else than in the handler so stick it in `MiscHandler.cpp` I guess.